pkg/drivers/kic/oci/oci.go

```diff
@@ -139,6 +140,36 @@ func hasMemorySwapCgroup() bool {
 	return memcgSwap
 }
 
+// HasCPUCFSPeriod checks whether it is possible to set cpu cfs period for cgroup.
+func HasCPUCFSPeriod() bool {
+	cpuCfsPeriod := true
+	if runtime.GOOS == "linux" {
```

Comment:
for non-Linux `HasCPUCFSPeriod()` always returns `true`. It it's by design, let's change then name. 
"CFS" make sense only in Linux context